### PR TITLE
feat: Add ChEMBL and Mondo

### PR DESCRIPTION
**Pull request by jsstevenson**
These are our two current problem children.

* ChEMBL: downloads via HTTP. I think FTP is probably faster, but I've been ~unable to maintain an FTP connection with the EMBL FTP server at the hospital, so I think it's easier to just go with HTTP, particularly since they don't do too many updates.
* Mondo: Includes some extra methods to support trying previous versions. The most recent Mondo release is unparseable by our OWL reader, and while I think they're going to make a change that resolves this, I don't think they view it as an urgent or critical fix (the problem is really with `owlready2`, not with their data output). 

There's a little bit of repeated code here. I think it'd be better to hold off on consolidating methods until we've seen what it looks like to implement this for every source that we're interested in.

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  - id: detect-private-key
+  - id: trailing-whitespace
+  - id: end-of-file-fixer
+- repo: https://github.com/psf/black
+  rev: 23.9.1
+  hooks:
+    - id: black
+      language_version: python3.11
+- repo: https://github.com/astral-sh/ruff-pre-commit
+  rev: v0.0.292
```
Comment (korikuzma) [inline .pre-commit-config.yaml:16]: Ruff made a [v0.1.0](https://github.com/astral-sh/ruff/releases/tag/v0.1.0) release

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v1.4.0
+  hooks:
+  - id: check-added-large-files
+    args: ['--maxkb=1500']
+  - id: detect-private-key
+  - id: trailing-whitespace
+  - id: end-of-file-fixer
+- repo: https://github.com/psf/black
+  rev: 23.9.1
```
Comment (korikuzma) [inline .pre-commit-config.yaml:11]: [23.10.0](https://github.com/psf/black/releases/tag/23.10.0)

File: README.md [hunk 3]
```diff
@@ -1,2 +1,70 @@
-# WagsTools
-Data acquisition tools for Wagnerds
+# WagsTails
```
Comment (korikuzma) [inline README.md:1]: Where is our acronym

File: README.md [hunk 4]
```diff
@@ -1,2 +1,70 @@
...
+
+Data acquisition tools for Wagnerds.
+
+## Installation
+
+Install from PyPI:
+
+```shell
+python3 -m pip install wagstails
```
Comment (korikuzma) [inline README.md:10]: ? We can keep as is if you prefer it better

```suggestion
python3 -m pip install wags_tails
```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,70 @@
...
+## Configuration
+
+All data is stored within source-specific subdirectories of a designated WagsTails data directory. By default, this location is `~/.local/share/wagstails/`, but it can be configured by passing a Path directly to a data class on initialization, via the `$WAGSTAILS_DIR` environment variable, or via [XDG data environment variables](https://specifications.freedesktop.org/basedir-spec/basedir-spec-0.6.html).
+
+## Development
+
+Check out the repository:
+
+```shell
+git clone https://github.com/GenomicMedLab/WagsTails
```
Comment (korikuzma) [inline README.md:42]: Why not lower case with hyphen like our other projects?

```suggestion
git clone https://github.com/GenomicMedLab/wags-tails
```

File: README.md [hunk 6]
```diff
@@ -1,2 +1,70 @@
...
+
+```shell
+python3 -m pip install -e '.[dev,test]'
+pre-commit install
+```
+
+Check style:
+
+```shell
+black src/ tests/ && ruff check --fix src/ tests/
```
Comment (korikuzma) [inline README.md:63]: ?

```suggestion
black . && ruff check --fix .
```

**Review by korikuzma — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (jsstevenson) [reply README.md:10]: now that you mentioned it, this would've solved the "wag stools" problem

File: .pre-commit-config.yaml (hunk 1 above)
Comment (jsstevenson) [reply .pre-commit-config.yaml:16]: they move quick

**Pull request merged**